app/src/main/java/com/sunasterisk/moviedb_51/ui/home/HomeFragment.kt

```diff
@@ -89,6 +93,19 @@ class HomeFragment : Fragment(), TabLayout.OnTabSelectedListener,
     }
 
     private fun observeViewModel() = with(viewModel) {
+        showEmptyMovieRecent.observe(viewLifecycleOwner, Observer {
+            if (it) {
+                messageEmptyTextView.text = getString(R.string.message_empty_movies_recent)
+                binding.layoutEmptyData.visibility = View.VISIBLE
+            } else binding.layoutEmptyData.visibility = View.GONE
```

Comment:
layoutEmptyData này sao không observer 1 trạng thái riêng nhỉ, true/false 
vì nó lẫn với nhiều thằng khác 
```suggestion
            binding.layoutEmptyData.isVisible = it
            if (it) {
                messageEmptyTextView.text = getString(R.string.message_empty_movies_recent)
            } 
```